problem_builder/public/js/mentoring_standard_view.js

```diff
@@ -35,10 +35,13 @@ function MentoringStandardView(runtime, element, mentoring) {
             }
         }
 
+        // Data may have change, we have to re-validate.
+        validateXBlock();
+
```

Comment:
@itsjeyd I fixed the typo, thanks for noticing. This probably isn't necessary for answer blocks anymore (although it shouldn't do any harm), but I believe it still makes sense for other blocks.